coderd/database/queries/user_links.sql

```diff
@@ -14,18 +14,23 @@ FROM
 WHERE
 	user_id = $1 AND login_type = $2;
 
+-- name: GetUserLinksByUserID :many
+SELECT * FROM user_links WHERE user_id = $1;
+
 -- name: InsertUserLink :one
 INSERT INTO
 	user_links (
 		user_id,
 		login_type,
 		linked_id,
 		oauth_access_token,
+		oauth_access_token_key_id,
```

Comment:
nit: if you move new values to the bottom of user_links's columns, you don't need to change WHERE below (in `UpdateUserLinkedID`).

Reply:
👍  Fair point. I'm going to keep it this way so that the `xxx` and `xxx_key_id` columns are right beside each other so they're less easy to miss.